packages/plugin-meta-extractor/src/bin/run.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+import { extractPluginMeta } from '../meta-extractor';
+const fs = require('fs');
+
+const entryPath = process.argv[2];
+
+// Check if the first argument is present
+if (entryPath === undefined) {
+  console.error('Please provide the path for the module.ts / module.tsx as an argument.');
+  console.error('(E.g. "npx @grafana/plugin-meta-extractor ./src/module.ts")');
+  process.exit(1);
+}
+
+// Check if the file exists
+if (!fs.existsSync(entryPath)) {
+  console.error(`File not found: ${entryPath}`);
+  process.exit(1);
+}
+
+// Check if it is a module file
+const fileName = entryPath.split('/').pop();
+if (!fileName?.match(/^(module\.ts|module\.tsx)$/)) {
```

Comment:
nit: I've not checked this but the following should be able to match without need for duplicating the filename

```suggestion
if (!fileName?.match(/^(module\.tsx?)$/)) {
```

Reply:
Nice, thanks! 👍